ports/igraph/portfile.cmake

```
@@ -4,9 +4,9 @@
 #  - The release tarball contains pre-generated parser sources, which eliminates the dependency on bison/flex.
 
 vcpkg_download_distfile(ARCHIVE
-    URLS "https://github.com/igraph/igraph/releases/download/0.10.16/igraph-0.10.16.tar.gz"
-    FILENAME "igraph-0.10.16.tar.gz"
-    SHA512 f17b2777b8817618958f2b5505a8d066a45a25ac05ee646e3ae7e2e6d19c86d90aca3ad05866a03fb2545d21caa374d736f4f9b8c090c04da3bd88c632792fd8
+    URLS "https://github.com/igraph/igraph/releases/download/0.10.17/igraph-0.10.17.tar.gz"
+    FILENAME "igraph-0.10.17.tar.gz"
```

Maybe with the next port update.
```suggestion
    URLS "https://github.com/igraph/igraph/releases/download/${VERSION}/igraph-${VERSION}.tar.gz"
    FILENAME "igraph-${VERSION}.tar.gz"
```

---

Thank you, this is done now.